docs/developers/listener.md

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
```
---
title: Working With Events
---
```

I'd also want to cover how similar the annotation-based mechanism is to BungeeCord, Bukkit, and Sponge.